16_wordle/main_test.go

```diff
@@ -37,6 +40,52 @@ func TestContains(t *testing.T) {
 	}
 }
 
+func TestNewList(t *testing.T) {
+	tests := []struct {
+		content string
+		want    List
+	}{
+		{
+			content: "brown\nrossa\ncuppy",
+			want: List{
+				words: [][]byte{
+					[]byte("brown"), []byte("rossa"), []byte("cuppy"),
+				},
+			},
+		},
+		{
+			content: "rossa\ncuppy",
+			want: List{
+				words: [][]byte{
+					[]byte("rossa"), []byte("cuppy"),
+				},
+			},
+		},
+	}
+
+	for index, test := range tests {
+		t.Run("", func(t *testing.T) {
+			tmpDir := t.TempDir()
+			listPath := filepath.Join(tmpDir, "listPath")
+
+			err := os.WriteFile(listPath, []byte(test.content), 0666)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
```suggestion
				t.Fatal(err)
```
 So that the test fails if it can't create the file

Reply:
Done